apps/meteor/app/importer/server/classes/ImportDataConverter.ts

```diff
@@ -371,6 +371,8 @@ export class ImportDataConverter {
 				this.saveError(_id, e instanceof Error ? e : new Error(String(e)));
 			}
 		});
+
+		await Promise.all(promises);
```

Comment:
Wondering if we should use AllSettled here instead. So if any one of the users has some record that isn't valid and is throwing errors then only that particular user won't be imported. The rest of the users will still get imported this way.